test/renpy-font-lock-test.el

```diff
@@ -1,5 +1,9 @@
 ;;; renpy-font-lock-test.el ---                     -*- lexical-binding: t; -*-
 
+;;; Commentary:
+
+;; Font-lock unit tests. Tests below focus on narrow font-lock cases.
```

Comment:
Flymake note: `There should be two spaces after a period`.

Reply:
which version of emacs are you using?  I don't get this warning no matter what I do.

Reply:
oh, I see. I do (setq sentence-end-double-space nil) in my config. 

Do you have a preference here? We might want to force this through a project-wide .dir-locals.el

Reply:
My preference would be to enforce the default values, so double space at the end of a sentence and spellcheck using a US dictionary.